hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_hasher.py

```diff
@@ -13,7 +13,7 @@
 
 from hmalib import metrics
 from hmalib.models import PipelinePDQHashRecord
-from hmalib.common import get_logger
+from hmalib.common.logging import get_logger
```

Comment:
Make sure to do a grep or something similar to make sure you get all the occurences.

Reply:
mypy was catching imports correctly